apps/user-web/src/components/common/select/index.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import { BottomSheet } from '@components/common';
+import useBottomSheet from '@hooks/use-bottom-sheet';
+import BottomArrow from '@icons/bottom-arrow.svg';
```

Comment:
arrow 라기 보다는 chevron 아닐까요?
```suggestion
import LowerChevron from '@icons/lower-chevron.svg';
```